pkg/build/builder/sti.go

```diff
@@ -354,8 +362,9 @@ func buildEnvVars(build *api.Build) s2iapi.EnvironmentList {
 // the strategy's environment. There is no preference given to either lowercase
 // or uppercase form of the variable.
 func scriptProxyConfig(build *api.Build) (*s2iapi.ProxyConfig, error) {
-	httpProxy := ""
-	httpsProxy := ""
+	var httpProxy string
+	var httpsProxy string
```

Comment:
Ok, I don't against such changes. I can read any of the ways of defining variables. Just wanted to be sure that this is an intentional change and not the forgotten part after some refactoring.